geomstats/_backend/numpy/linalg.py

```diff
@@ -1,8 +1,11 @@
 """Numpy based linear algebra backend."""
 
 import autograd.numpy as np
+import autograd.scipy.linalg as asp
```

Comment:
Do we need scipy.linalg if it has been replaced by autograd's version?

Reply:
Yes, I had troubles with the log but it works now